test/template-tests/ConvertFrom-JSON.ps1

```diff
@@ -0,0 +1,61 @@
+﻿function ConvertFrom-Json
+{
+[CmdletBinding(HelpUri='https://go.microsoft.com/fwlink/?LinkID=217031', RemotingCapability='None')]
+param(

```

Comment:
let's add a switch to ignore comments or not...  Just as if we were doing this in the real cmdlet ;)